frontend/src/routes/(authenticated)/(dashboards)/admin/+page.svelte

```diff
@@ -122,10 +185,60 @@
               <td>
                 <FormatDate date={user.createdDate} />
               </td>
+            <td>
+                <button class="btn btn-ghost rounded" on:click={async () => {await openModal(user)}}><PencilIcon></PencilIcon></button></td>
```

Comment:
```suggestion
                <button class="btn btn-ghost btn-square rounded" on:click={async () => {await openModal(user)}}><PencilIcon></PencilIcon></button></td>
```

Actually, it would be even better if you used IconButton here, which would need adapting, because, you're right, these should be ghost-buttons, not outline-buttons 😀